### ISSUE TITLE
types.UID

### ISSUE THREAD

**Issue by kris-nova** [kris-nova/naml#57]
duffie was able to generate some code with undefined `types`

**Comment by fkautz** [comment]
I just did a poc with valast to get it to dynamically generate a map of imports when generating the AST. I integrated the poc with naml, and got this out of it:

```go
map[:true
    github.com/hexops/valast:true
    k8s.io/api/apps/v1:true
    k8s.io/api/core/v1:true
    k8s.io/apimachinery/pkg/apis/meta/v1:true
    k8s.io/apimachinery/pkg/types:true
    k8s.io/apimachinery/pkg/util/intstr:true]
``

I can try to upstream the changes, or set up a fork if they don't accept them.

Would you be interested in this as a path to deal with imports?

**Comment by fkautz** [comment]
More specifically, I'm happy to do the work to wire it up. :)

**Comment by kris-nova** [comment]
yes please! if you PR this i will merge it

**Comment by fkautz** [comment]
Expect a patch tonight.

**Comment by fkautz** [comment]
This should be resolved when #59 is merged.
